### PR TITLE
JAMES-2274 Per SHA-1 packaging

### PR DESCRIPTION
**Pull request by chibenwa**

### CODE REVIEW

**Comment by chibenwa** [comment]
test this please

File: dockerfiles/packaging/guice/cassandra/package.sh [hunk 1]
```diff
@@ -2,60 +2,40 @@
```
Comment (aduprat) [inline dockerfiles/packaging/guice/cassandra/package.sh]: Can you rename this script to _buildAndPackage.sh_?

**Comment by aduprat** [comment]
Two comments:
- needs a rebase
- are'nt we waiting for a cassandra-ldap product?

**Comment by chibenwa** [comment]
- I forgot to push the rebase
 - This PR, is I believe orthogonal to a LDAP product, so I believe if we want one, we need to create another ticket + PR

**Comment by aduprat** [comment]
- Thx for the rebase
- I think we should do the LDAP product right after merging this pull request, as it's the one our customer is waiting for

**Comment by chibenwa** [comment]
Merged

**Pull request closed without merging**